### PR TITLE
WSTEAM1-1157 - Next.js app ATI analytics object fix

### PR DESCRIPTION
**Pull request by amoore108**
Part of https://jira.dev.bbc.co.uk/browse/WSTEAM1-1157

Overall changes
======
- Extracts ATI analytics from the new `metadata.atiAnalytics` object if available
- This will enable backwards compatiblity whilst the BFF changes are made to return analytics values in the new `metadata.atiAnalytics` structure, falling back to using the values from the root `pageData` object if not: https://github.com/bbc/simorgh/blob/76620a8856e726ddf5839d7916266c1ac8079c46/src/app/components/ATIAnalytics/params/index.ts#L152

Testing
======
- Run this branch locally
- Open a Live page, e.g. http://localhost:7081/burmese/live/cx8jgp1zxylt?renderer_env=live
- Check the Chrome dev tools, under the 'Fetch/XHR' tab
- Confirm that ATI beacons continue to send as before

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/_app.page.tsx [hunk 1]
```diff
@@ -67,6 +67,8 @@ export default function App({ Component, pageProps }: Props) {
     isUK,
   } = pageProps;
 
+  const { metadata: { atiAnalytics } = {} } = pageData;
```
Comment (HarveyPeachey) [inline ws-nextjs-app/pages/_app.page.tsx]: I don't think the live page response has a metadata object, the atiAnalytics just lives in the pageData

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/_app.page.tsx]: Ahh sorry didn't see your FABL PR

**Review by HarveyPeachey — APPROVED** [review]

File: src/app/components/ATIAnalytics/types.ts [hunk 2]
```diff
@@ -26,10 +26,7 @@ export interface PageData {
       pageTitle?: string;
       producer?: string;
     };
-    atiAnalytics?: {
-      producerId?: string;
-      chapter?: string;
-    };
+    atiAnalytics?: ATIData;
```
Comment (karinathomasbbc) [inline src/app/components/ATIAnalytics/types.ts:46]: I initially couldn't find `ATIData`, as I was searching above this code / in the imports to see where it was declared. Turns out it is declared below & typescript doesn't seem to care about the order of declaration in the types file. Would it be possible to move the declaration of `ATIData` above where it's used, just for ease of reading & consistency with other types?

**Review by emilysaffron — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/components/ATIAnalytics/types.ts (hunk 2 above)
Comment (amoore108) [reply src/app/components/ATIAnalytics/types.ts:46]: Yea it doesn't care about the order. I've re-ordered that particular type now.

Comment (karinathomasbbc) [reply src/app/components/ATIAnalytics/types.ts:46]: Cool, thanks! 😎

**Pull request merged**